src/test/java/test/selenium_tests/CartPageTests.java

```diff
@@ -0,0 +1,49 @@
+package test.selenium_tests;
+
+import core.utils.WaiterUtils;
+import org.junit.Test;
+import pages.*;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class CartPageTests extends SeleniumTestsBase {
+
+    private SearchResultPage searchResultPage = new SearchResultPage();
+    private HeaderBlock header = new HeaderBlock();
+    private ItemPage itemPage = new ItemPage();
+    private CartPage cartPage = new CartPage();
+
+    @Test
+    public void checkSubtotalValuesInCart() throws InterruptedException {
+        List<Double> listOfPrices = new ArrayList<>();
+        List<String> listOfSearchItems = Arrays.asList("Sweese Porcelain Tea Mug with Infuser and Lid"
```

Comment:
I'd suggest make search criterias simplier, like `mug`

Reply:
made it simplier